app.js

```diff
@@ -2,87 +2,75 @@ const storageKey = 'awesomeBooks';
 const currentFormTitle = document.querySelector('#title');
 const currentFormAuthor = document.querySelector('#author');
 const currentFormBookAddButton = document.querySelector('#book-add-button');
-const arrBooks = [];
 
-function removeBook(id) {
-  for (let i = 0; i < arrBooks.length; i += 1) {
-    if (arrBooks[i].id === id) {
-      arrBooks.splice(i, 1);
-    }
+class BookShelf {
+  constructor() {
+    this.arrBooks = [];
   }
-}
-
-function addBook() {
-  let maxId = 0;
-  arrBooks.forEach((book) => {
-    if (book.id > maxId) {
-      maxId = book.id;
-    }
-  });
 
-  const newBook = {
-    id: maxId + 1,
-    title: currentFormTitle.value,
-    author: currentFormAuthor.value,
-  };
-  arrBooks.push(newBook);
-}
+  addBook(newTitle, newAuthor) {
+    const timeNow = new Date();
+    const newBook = {
+      id: timeNow.getTime(),
+      title: newTitle,
+      author: newAuthor,
+    };
 
-function saveDataToLocalStorage() {
-  localStorage.setItem(storageKey, JSON.stringify(arrBooks));
-}
+    this.arrBooks.push(newBook);
+  }
 
-function addElement(elementType, parent, className) {
-  const element = document.createElement(elementType);
-  element.classList.add(className);
-  parent.appendChild(element);
-  return element;
-}
+  removeBook(id) {
+    this.arrBooks = this.arrBooks.filter((book) => book.id !== id);
+  }
```

Comment:
- [ ] Since I was unable to add a book I was unable to verify that the book removal function works. Kindly ensure that as well.